lib/katello/tasks/repository.rake

```diff
@@ -59,6 +59,29 @@ namespace :katello do
     end
   end
 
+  desc "Change the download policy of all repos. Specify DOWNLOAD_POLICY=policy"
```

Comment:
@daviddavis is it possible to specify the different options for `DOWNLOAD_POLICY` in the description?